src/direct/pcm.rs

```diff
@@ -20,12 +20,12 @@ For an example of how to use this mode, look in the "synth-example" directory.
 */
 
 use libc;
-use std::{mem, ptr, fmt, cmp};
+use core::{mem, ptr, fmt, cmp};
 use crate::error::{Error, Result};
-use std::os::unix::io::RawFd;
+use core::ffi::c_int;
```

Comment:
As for RawFd. Would it be possible to declare it somewhere like

```
#feature(std)
type RawFd = std::os::unix::io::RawFd;
#feature(no-std)
type RawFd = core::ffi::c_int;
```

...and then use that definition of `RawFd` everywhere else?
Makes the code more readable I believe.